index.html

```diff
@@ -14,10 +14,11 @@
     <h1> Number<span class='guesser-title'>Guesser</span></h1>
   </header>
   <section class="inputArea">
-    <input class="userInput" placeholder="Enter Your Guess" type="number"></input>
+    <input class="userInput" placeholder="Enter Your Guess" type="number">
     <button class="guess">
       <p class="button-text">Guess</p>
     </button>
+    <!-- <button>Guess</button> -->
```

Comment:
Review this commented out line of code for a guide to refactor your buttons. You don't need to nest another `p` element inside a `button` element.